recipes/dcw-gmt/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "dcw-gmt" %}
+{% set version = "1.1.2" %}
+{% set sha256 = "f719054f8d657e7b10b5182d4c15bc7f38ef7483ed05cdaa9f94ab1a0008bfb6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: ftp://ftp.soest.hawaii.edu/dcw/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+test:
+  commands:
+    - test -f $PREFIX/share/gcw-gmt/LICENSE.TXT                             # [unix]
+    - test -f $PREFIX/share/gcw-gmt/{{ name }}.nc                           # [unix]
+    - if not exist %PREFIX%\\Library\\share\\gcw-gmt\\LICENSE.TXT exit 1    # [win]
+    - if not exist %PREFIX%\\Library\\share\\gcw-gmt\\{{ name }}.nc exit 1  # [win]
```

Comment:
@mhearne-usgs looks like we didn't notice that the package name is wrong (`gcw` instead of `dcw`). Use `{{ name }}`  instead to avoid errors in meta.yml